defi/Compound/Compound清算机器人设计.md

```diff
@@ -0,0 +1,71 @@
+# Compound清算机器人设计
+
+清算机器人的目的是为了帮助协议清算有风险的债务，同时获得协议奖励。
```

Comment:
you can hold a small talk for this subject, for example: how to code, or the senario which the bot can work for?